dags/resources/stages/parse/table_definitions/balancer/V2_BalancerPool_event_PoolCreated.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "pool",
+                    "type": "address"
+                }
+            ],
+            "name": "PoolCreated",
+            "type": "event"
+        },
+        "contract_address": "SELECT '0x8df6efec5547e31b0eb7d1291b511ff8a2bf987c' UNION ALL SELECT '0x8b7854708c0c54f9d7d1ff351d4f84e6de0e134c' UNION ALL SELECT '0x2e93403c675ccb9c564edf2dc6001233d0650582' UNION ALL SELECT '0x74b5d15f316a4939da6349169d266b0db0820c20' UNION ALL SELECT '0x9ac3e70db606659bf32d4bdfbb687ad193fd1f5b' UNION ALL SELECT '0xf4c6bb0cffd9be0be38a222e8d6833344aee5cc1' UNION ALL SELECT '0x8e9aa87e45e92bad84d5f8dd1bff34fb92637de9' UNION ALL SELECT '0xb9f8ab3ed3f3acba64bc6cd2dca74b7f38fd7b88' UNION ALL SELECT '0x5dd94da3644ddd055fcf6b3e1aa310bb7801eb8b' UNION ALL SELECT '0x5f43fba61f63fa6bff101a0a0458cea917f6b347' UNION ALL SELECT '0xc66ba2b6595d3613ccab350c886ace23866ede24' UNION ALL SELECT '0xcc508a455f5b0073973107db6a878ddbdab957bc' UNION ALL SELECT '0xb8f7ad3af5238289b5ced8df3b7bcc41a936a458' UNION ALL SELECT '0xe88628700eae9213169d715148ac5a5f47b5dcd9' UNION ALL SELECT '0x67d27634e44793fe63c467035e31ea8635117cd4' UNION ALL SELECT '0xb5a0a6bcecb2988bb348c2546bba9c2bd9a04a1e' UNION ALL SELECT '0xb7561f547f3207edb42a6afa42170cd47add17bd' UNION ALL SELECT '0x73b7ead3c904beb91f5c04ac6207dd8074cb521e' UNION ALL SELECT '0xdba127fbc23fb20f5929c546af220a991b5c6e01' UNION ALL SELECT '0xa1edd4e98e353cad59ab080ca25e1b856bc5e30b' UNION ALL SELECT '0x321aed5d4c558d54e3694fc13689962b3b0c06ee' UNION ALL SELECT '0xdc6d62ae091ea77331542042a72e5e38b188837c' UNION ALL SELECT '0x6a0ac04f5c2a10297d5fa79fa6358837a8770041' UNION ALL SELECT '0x0f3e0c4218b7b0108a3643cfe9d3ec0d4f57c54e' UNION ALL SELECT '0x48767f9f868a4a7b86a90736632f6e44c2df7fa9' UNION ALL SELECT '0x8de8895ddd702d9a216e640966a98e08c9228f24' UNION ALL SELECT '0x9e629751b3fe0b030c219e567156adcb70ad5541' UNION ALL SELECT '0xa5bf2ddf098bb0ef6d120c98217dd6b141c74ee0' UNION ALL SELECT '0x2ebe41e1aa44d61c206a94474932dadc7d3fd9e3' UNION ALL SELECT '0x267a1ece57fba089ecf39d21b693b7d63b9a6a74' UNION ALL SELECT '0x67a25ca2350ebf4a0c475ca74c257c94a373b828' UNION ALL SELECT '0x7d833fef5bb92ddb578da85fc0c35cd5cc00fb3e' UNION ALL SELECT '0xba1b4a90bad57470a2cba762a32955dc491f76e0' UNION ALL SELECT '0xd7fad3bd59d6477cbe1be7f646f7f1ba25b230f8' UNION ALL SELECT '0x27fe2bfbb6be96d64db0e741078a1f29aa20226b' UNION ALL SELECT '0x751a0bc0e3f75b38e01cf25bfce7ff36de1c87de' UNION ALL SELECT '0xf9ac7b9df2b3454e841110cce5550bd5ac6f875f' UNION ALL SELECT '0xe061bf85648e9fa7b59394668cfeef980aec4c66' UNION ALL SELECT '0x85a80afee867adf27b50bdb7b76da70f1e853062' UNION ALL SELECT '0xb08e16cfc07c684daa2f93c70323badb2a6cbfd2'",
```

Comment:
This does not pass the syntax [test](https://github.com/blockchain-etl/ethereum-etl-airflow/actions/runs/4471450975/jobs/7856358236?pr=538). Can try using `UNNEST` like [here](https://github.com/blockchain-etl/ethereum-etl-airflow/blob/3dc3298f260d8fdcc229dddab0f2138a97d55661/dags/resources/stages/parse/table_definitions/niftygateway/BuilderShop_call_createNewBuilderInstance.json#L42)